convene-web/app/views/workspaces/_room_card.html.erb

```diff
@@ -1,4 +1,4 @@
-<li class="col-span-1 bg-white rounded-lg shadow">
+<li class="col-span-1 bg-white rounded-lg shadow" name="<%= room.name %>" >
```

Comment:
I would recommend using [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) over [`name`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#Attribute_list) as the `name` attribute is intended to communicate form field attributes.